src/components/ArchivePanel.astro

```diff
@@ -68,18 +68,22 @@ function formatTag(tag: string[]) {
 <div class="card-base px-8 py-6">
     {
         groups.map(group => (
-            <div>
+            <div key={group.year}>
```

Comment:
Could you please explain the purpose of adding this attribute?

Reply:
I guess this is unnecessary in Astro since there is no dynamic element updating. If you have time, could you help me remove these parameters? I'm worried that I might forget why they are here in the future.🥲